_connect-files/api/objects/form-properties/sources/saas/onfleet-object.md

```diff
@@ -0,0 +1,59 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+## PLEASE REMOVE COMMENTS WHEN FINISHED
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-onfleet-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Onfleet Source Form Property"
+api-type: "platform.onfleet"
+display-name: "Onfleet"
+
+source-type: "saas"
+docs-name: "onfleet"
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+# uses-common-fields: true/false
+# See these fields in _data/connect/common/all-sources.yml
+# May also include applicable fields in _data/connect/common/all-sources.yml
+
+object-attributes:
+  - name: "api_key"
+    type: "string"
+    required: true
+    description: |
+      The user's {{ form-property.display-name }} API key. Refer to the [{{ form-property.display-name }} documentation]({{ doc-link }}) for instructions on creating this credential.
```

Comment:
Should `{{ doc-link }}` be pointing to an Onfleet doc page?

Reply:
It points to our docs for creating the API key.

Reply:
Ohhh, I see.